torch/nn/quantized/modules/conv.py

```diff
@@ -0,0 +1,104 @@
+# coding=utf-8
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import torch
+from torch.nn.modules.utils import _pair
+from torch.nn.quantized import functional as qF
+
+from torch.nn.modules.conv import _ConvNd
+
+"""Computes the output shape given convolution parameters."""
+def _conv_output_shape(input_size, kernel_size, padding, stride, dilation,
+                       output_padding=0):
+    return np.floor((input_size + 2 * padding - kernel_size - (kernel_size - 1)
+                    * (dilation - 1)) / stride) + 2 * output_padding + 1
+
+
+class Conv2d(_ConvNd):
+    def __init__(self, in_channels, out_channels, kernel_size, stride=1,
+                 padding=0, dilation=1, groups=1,
+                 bias=True, padding_mode='zeros'):
+        if padding_mode == 'circular':
+            raise NotImplementedError("Circular padding is not implemented!")
+        stride = _pair(stride)
+        padding = _pair(padding)
+        dilation = _pair(dilation)
+        kernel_size = _pair(kernel_size)
+        transposed = False
+        output_padding = _pair(0)
+        super(Conv2d, self).__init__(in_channels=in_channels,
+                                     out_channels=out_channels,
+                                     kernel_size=kernel_size,
+                                     stride=stride,
+                                     padding=padding,
+                                     dilation=dilation,
+                                     transposed=transposed,
+                                     output_padding=output_padding,
+                                     groups=groups,
+                                     bias=True,
+                                     padding_mode=padding_mode)
+        del self.weight
+        del self.bias
+
+        qweight = torch._empty_affine_quantized(
+            [out_channels, kernel_size[0], kernel_size[1],
+             in_channels // self.groups],
+            scale=1, zero_point=0, dtype=torch.qint8)
+        qbias = torch._empty_affine_quantized([out_channels],
+                                              scale=1, zero_point=0,
+                                              dtype=torch.qint32)
+        self.register_buffer('_packed_weight',
+                             torch.ops.quantized.fbgemm_conv_prepack(qweight, self.groups))
+        self.register_buffer('bias', qbias)
+        self.register_buffer('_scale', torch.Tensor([1]))
```

Comment:
types for scale and zero_point should be torch.double and torch.long actually, also can you make sure linear.py is the same as well? Thanks